boards/opencm904/include/board.h

```diff
@@ -55,11 +55,20 @@ extern "C" {
 /** @} */
 
 /**
- * @brief   User button
+ * @brief User button
  */
 #define BTN_B1_PIN          GPIO_PIN(PORT_C, 15)
 
 /**
+ * @brief Pin used to switch RX and TX mode for the Dynamixel TTL bus
```

Comment:
my bad, it seems that this is for the Dynamixel TTL bus which is available on that board.